nvi-commons/src/main/java/no/sikt/nva/nvi/common/db/CandidateDao.java

```diff
@@ -336,18 +336,6 @@ public static Builder builder() {
             return new Builder();
         }
 
-        public String instanceType() {
-            return instanceType;
-        }
-
-        public Instant modifiedDate() {
-            return modifiedDate;
-        }
-
-        public Instant createdDate() {
-            return createdDate;
-        }
-
```

Comment:
Nice, I think it would actually would be prettier to put all this objects to own java classes, so we do not have a java class over 700 lines.

Reply:
Yes, this could probably be split up. I suspect some of the code is unused/unneeded as well, such as the hashcode implementation. Should do it in a separate PR though ;-)